config/initializers/secret_token.rb

```diff
@@ -1,5 +1,5 @@
 if Rails.env.production?
-  IndyhackersRails::Application.config.secret_token = ENV['OMG_SECRET_TOKEN']
+  IndyhackersRails::Application.config.secret_key_base = ENV['OMG_SECRET_TOKEN']
```

Comment:
This may or may not work in current Rails? Usually it looks for a `RAILS_SECRET_KEY_BASE` or some environment variable named similarly. I **think** this will still work.